### PR TITLE
Refactor address page to formik and use a modal instead

### PR DESCRIPTION
**Pull request by ranzwertig**
# Definition of Done

- [x] The code does at least what is defined in the ticket and does not affect any other functionality
- [x] The PR has a meaningful title
- [x] There are no 'hacks' or shortcuts refactoring is preferred
- [x] The happy case of the app was tested at least once manually

### CODE REVIEW

**Comment by dreampulse** [comment]
# UX Issues

- [ ] Fix merge conflict
- [ ] Update modal design (rounded cornerns, shadow)
- [ ] Color and margin-bottom of the headline

File: package.json [hunk 1]
```diff
@@ -125,6 +125,7 @@
     "debounce-promise": "^3.0.1",
```
Comment (dreampulse) [inline package.json]: Die `redux-form` kann aus der `package.json` entfernt werden.
Ebenso in `reducer/index.ts`

File: src/app/container/util/form.js [hunk 2]
```diff
@@ -1,25 +1,28 @@
...
+      {...props}
       modifiers={modifiers}
-      {...props.input}
-      {...omit(props, ['input', 'meta'])}
+      checked={Boolean(field.value)}
       onChange={e => {
-        props.input.onChange(e)
         if (props.onChangeValue && typeof props.onChangeValue === 'function') {
-          props.onChangeValue(props.input.value, props.input.name)
+          props.onChangeValue(!field.value, field.name)
```
Comment (dreampulse) [inline src/app/container/util/form.js]: Vlt kann man das `onChangeValue` los werden

File: src/app/container/review-order-page.js [hunk 3]
```diff
@@ -80,84 +80,107 @@ const ReviewOrderPage = ({
...
+            <>
+              <Headline modifiers={['l', 'minor']} label="Shipping Address" />
+              <Button label="Add Address" onClick={() => openAddressFormModal()} />
+            </>
+          )}
+          {user && (
+            <>
+              <Headline
+                modifiers={['minor', 'l']}
+                label={
```
Comment (dreampulse) [inline src/app/container/review-order-page.js]: Das hier schmeißt ne Warnung. Das muss allerdings in der `Headline`-Componente gefixt werden.
```
    <>
      {finalIcon ? <Icon key="icon" source={finalIcon} /> : null}
      {label}
    </>
```

**Comment by dreampulse** [comment]
`http://localhost:8000/v3/user`:
`useDifferentBillingAddress: null` <- Das muss immer ein bool sein `Invalid request body: Invalid type: null (expected boolean)`.

**Comment by ranzwertig** [comment]
@Dansken @henry-all3dp please test and merge

**Pull request merged**